src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,17 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
```

Comment:
redundant empty line

Reply:
When i rremove this empty line, i have an error of checkstyle: METHOD_DEF' should be se parated from previous statement. [EmptyLineSeparator]